code/game/objects/items/devices/radio/headset.dm

```diff
@@ -223,13 +223,37 @@
 /obj/item/device/radio/headset/headset_cent/commander
 	keyslot = new /obj/item/device/encryptionkey/heads/captain
 
-/obj/item/device/radio/headset/headset_cent/alt
+/obj/item/device/radio/headset/headset_cent/ert
 	name = "\improper Centcom bowman headset"
 	desc = "A headset especially for emergency response personnel. Protects ears from flashbangs. \nTo access the centcom channel, use :y."
 	icon_state = "cent_headset_alt"
 	item_state = "cent_headset_alt"
 	keyslot = null
 
+/obj/item/device/radio/headset/headset_ert_commander
+	name = "\improper Centcom headset"

```

Comment:
I have to agree with ike here if only because that's how it was before. Keep it as it was before and make a second pr for changing case to have an actual discussion on it.